.github/workflows/preview_build.yml

```
@@ -59,5 +59,5 @@ jobs:
       uses: peter-evans/create-or-update-comment@v2
       with:
        issue-number: ${{github.event.issue.number}}
-       body: "This PR is being built with Sphinx and can be previewed at: /pr/${{github.event.issue.number}}"
+       body: "This PR is has been built with Sphinx and can be previewed at: /pr/${{github.event.issue.number}}"
```

Should we include a full link here?

```suggestion
       body: "This PR is has been built with Sphinx and can be previewed at: https://fortran-lang.github.io/webpage/pr/${{github.event.issue.number}}"
```

This should work even with when we are using a custom domain name later, since the GitHub pages will automatically redirect.

---

We can surely do that sir, but as we are going to shift to fortran-lang.org then at that time we would have to rechange the url.

---

Maybe we can extract the repository name from the GitHub context and infer the GitHub pages name.